src/Category/Applicative.agda

```diff
@@ -12,29 +12,29 @@
 module Category.Applicative where
 
 open import Level using (Level; suc; _⊔_)
-open import Data.Unit
+open import Data.Unit.Polymorphic
 open import Category.Applicative.Indexed
 
 private
   variable
     f : Level
 
 RawApplicative : (Set f → Set f) → Set (suc f)
-RawApplicative F = RawIApplicative {I = ⊤} λ _ _ → F
+RawApplicative {i} F = RawIApplicative {I = ⊤ {i}} λ _ _ → F
```

Comment:
It's a bit confusing to call the level `f` in the type and then `i` in the definition! Same issue below, but not in the other similar files.

Reply:
Yes, that was a mistake.  Plus, here in particular, there is very little point in making it level polymorphic.  I'll revert the changes to the various `Monad` functions.